sdk/servicebus/service-bus/test/internal/batchReceiver.spec.ts

```diff
@@ -886,41 +871,95 @@ describe("Batching Receiver", () => {
         refreshConnectionCalled.should.be.greaterThan(0, "refreshConnection was not called.");
       });
 
-      it("returns messages if drain is in progress (receiveAndDelete)", async function(): Promise<
+      //
+      // TODO: this is the one that appears to consistently fail.
+      //
+      //
+      //
+      //
+
+      // it.only("returns messages if drain is in progress (receiveAndDelete)", async function(): Promise<
+      //   void
+      // > {
+      it.only("returns messages if drain is in progress (receiveAndDelete)", async function(): Promise<
         void
       > {
         // Create the sender and receiver.
+
+        testLogger.info("Before the test");
+
         await beforeEachTest(noSessionTestClientType, "receiveAndDelete");
 
         // The first time `receiveMessages` is called the receiver link is created.
         // The `receiver_drained` handler is only added after the link is created,
         // which is a non-blocking task.
-        await receiver.receiveMessages(1, { maxWaitTimeInMs: 1000 });
-        const receiverContext = (receiver as ServiceBusReceiverImpl)["_context"];
+
+        testLogger.info(
+          "Receiving a single message to warm up receiver (there isn't one, so this should just time out)"
+        );
+
+        await receiver.receiveMessages(1);
```

Comment:
Curious why you removed the maxWaitTimeInMs. Was this method returning before the link initialization completed sometiems? (Seems reasonable)

Reply:
Generally, just removing them and using our 'default' experience. If it can't survive some slow connections then we need to think about it some more.